.ci/cfg-indent-check.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+
+import os
+import re
+import sys
+
+
+def cfg_indent_check():
+    print("Running cfg indent check...")
+    exit_code = 0
+    for file in sys.argv[1:]:
+        status_code = 0
+        cfg_path = os.path.abspath(os.path.join(os.path.dirname(__file__),
+                                                os.pardir, file))
+        with open(cfg_path, "r") as f:
+            contents = f.read().splitlines()
+            for num, line in enumerate(contents, 1):
```

Comment:
```suggestion
        with open(cfg_path, "r") as f:
            for num, line in enumerate(f, 1):
```
Actually, `f` is <class '_io.TextIOWrapper'> which herites [IOBase](https://docs.python.org/3/library/io.html#io.IOBase)
And the `IOBase` has the method `__iter__`, so I think we can iterate the value to improve the algorithm.

Reply:
I tried but seems next(f) will not strip '\n', this means the first check will always fail, so splitlines() would be better. If I handle `\n` in the next line with `line.rstrip()`, at the final line, we hard to check if we have a final newline. So I am considering keeping the current usage, even if it will take more time.